### PR TITLE
Add support for extra Jinja templates

### PR DESCRIPTION
**Pull request by yinzara**
* Add support for extra Jinja templates (#57)
* Bump minor version

### CODE REVIEW

File: src/sabledocs/sable_config.py [hunk 1]
```diff
@@ -36,13 +38,16 @@ def __init__(self, config_file_path):
 
         if path.exists(config_file_path):
             print(f"Configuration found in {config_file_path}")
-            with open(config_file_path, mode='rb') as config_file:
+            with (open(config_file_path, mode='rb') as config_file):
```
Comment (markvincze) [inline src/sabledocs/sable_config.py:41]: Is there a specific reason for surrounding this expression with parens? I don't see this pattern in the Python docs, nor do we use it with `if`, `for`, etc.
Not a big deal, just curious.

File: src/sabledocs/sable_config.py [hunk 2]
```diff
@@ -36,13 +38,16 @@ def __init__(self, config_file_path):
...
                 config_values = tomllib.load(config_file)
 
                 self.module_title = config_values.get('module-title', self.module_title)
                 self.input_descriptor_file = config_values.get('input-descriptor-file', self.input_descriptor_file)
                 self.template = config_values.get('template', self.template).rstrip("/\\")
                 self.template_path = config_values.get('template-path', self.template_path).rstrip("/\\")
+                self.extra_template_path = config_values.get('extra-template-path', self.extra_template_path
+                                                             ).rstrip("/\\")
```
Comment (markvincze) [inline src/sabledocs/sable_config.py:49]: Can we join these two lines, or if it'd be too long, then indent like this?
```
                self.extra_template_path = config_values.get(
                    'extra-template-path',
                    self.extra_template_path).rstrip("/\\")
```

File: src/sabledocs/sable_config.py (hunk 1 above)
Comment (yinzara) [reply src/sabledocs/sable_config.py:41]: No, this was totally a mistake. I have no idea how I did that. Let me remove. Do you want me to rebase it away or are you ok with an extra commit referencing the review?

Comment (markvincze) [reply src/sabledocs/sable_config.py:41]: Thanks, whichever is easiest for you is fine, I haven't been too consistent with rebasing/squashing vs merging all the commits, so it's okay either way.

**Comment by markvincze** [comment]
Thanks for the submission, cool stuff!

Looks good, only had those two small comments, I'll merge after you commmit.

File: src/sabledocs/sable_config.py (hunk 2 above)
Comment (yinzara) [reply src/sabledocs/sable_config.py:49]: You got it.

**Comment by yinzara** [comment]
You're all set! I amended the prior commit so there is only one commit for the PR.

**Pull request merged**